tests/test_integration.py

```diff
@@ -113,12 +113,14 @@ def test_proportion_of_demands_met(self):
             self.edf_algo_true_analysis_dict['proportion_of_demands_met'])
 
     def test_current_unbalance_nema(self):
-        np.testing.assert_allclose(
-            acnsim.current_unbalance(self.sim, ['Primary A', 'Primary B', 'Primary C']),
-            np.array(self.edf_algo_true_analysis_dict['primary_current_unbalance_nema']))
-        np.testing.assert_allclose(
-            acnsim.current_unbalance(self.sim, ['Secondary A', 'Secondary B', 'Secondary C']),
-            np.array(self.edf_algo_true_analysis_dict['secondary_current_unbalance_nema']))
+        with self.assertWarns(RuntimeWarning):
```

Comment:
Add a comment about why a RuntimeWarning should be excepted.

Reply:
My rationale in adding this warning was more for a "regression test," i.e. so we'd be alerted if this warning were no longer raised due to a dependency update or updated current unbalance definition. I'm of the mind that regression tests should try to track *any* changes in behavior, including changes in which warnings were raised. I don't know the technical reason as to why this error is raised, but I can make an issue to look into it. For now, I'll add a comment describing the rationale for the warning check.

Reply:
I added a comment pointing to issue #57, where we could discuss reasons and a fix if there is one.

Reply:
Ok. I think the comment is fine. Just wanted to document why we thought a RuntimeWarning would happen.